hw_diag/views/diagnostics.py

```diff
@@ -111,3 +114,46 @@ def version_information():
     }
 
     return response
+
+
+@DIAGNOSTICS.route('/v1/add-gateway-txn', methods=['POST'])
+def add_gateway_txn():
+    """
+    Generates an add_gateway_txn if a destination name and wallets are defined and valid.
+    Diagnostics report will be in the format below if successful.
+
+    https://docs.helium.com/mine-hnt/full-hotspots/become-a-maker/hotspot-integration-testing/#generate-an-add-hotspot-transaction
+    {
+        errors: [],
+        DESTINATION_ADD_GATEWAY_TXN_KEY: {
```

Comment:
@posterzh - The actual payload would contain the actual key name `shipping_destination_add_gateway_txn` here right? because `DESTINATION_ADD_GATEWAY_TXN_KEY` itself is a constant from `hm_pyhelper.constants.shipping`. Can you confirm this?

Reply:
Correct. `DESTINATION_ADD_GATEWAY_TXN_KEY` comes from `hm_pyhelper.constants.shipping` and its value is `shipping_destination_add_gateway_txn`.